### DISCUSSION TITLE
アップロード用のSDKをインポートすると、沢山の参照エラーが起きる

### DISCUSSION THREAD

**Discussion by Nekokojpn** [pixiv/vroid-sdk-developers#48] [Q&A]
こんにちは、お世話になります。

先日アップロード用SDKをご提供いただき、
アップロード用のSDK(VRoidSDKUploadExtension-v0.1.1)をインポートしたところ、下記画像のように沢山のエラーが起きます。
![Screen Shot 2022-07-05 at 18 02 55](https://user-images.githubusercontent.com/43212800/177292017-0c2cf34a-8b4d-4449-96dc-7215c7dd0128.png)

察するに、VRoidSDKUploadExtension-v0.1.1.unitypackageだけでは動作しないのではと考えております。

他にもSDKが必要なのでしょうか？
初歩的な質問で申し訳ございません。

回答いただけたら幸いです。
よろしくお願いいたします。

Unity 2020.3.25f1

**Comment by nyamadan** [comment]
お世話になっております、 [nyamadan](https://github.com/pixiv/vroid-sdk-developers/discussions/3#discussioncomment-1877844)です。

ご指摘の通り、通常のSDKが必要となります。

メールにて案内しましたURLよりVRoidSDK-v0.1.1.zipをダウンロードし、インポートをお願いいたします。
